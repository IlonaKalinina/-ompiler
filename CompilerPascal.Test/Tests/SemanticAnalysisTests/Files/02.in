program Test_02;
type
a = integer;
a = string;
var
b : a;
begin
end.