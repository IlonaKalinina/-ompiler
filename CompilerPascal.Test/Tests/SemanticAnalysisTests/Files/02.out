(4,11) ERROR: Duplicate identifier "a"
